packages/e2e_tests/cypress/e2e/get-updates-sign-up.cy.js

```diff
@@ -0,0 +1,35 @@
+import { PO_GetUpdates } from '../pageObject/Get-updates/PO_GetUpdates';
+
+const getUpdates = new PO_GetUpdates();
+
+describe('User subscribes to get updates about a project', () => {
```

Comment:
```suggestion
describe('when user subscribes to get updates about a project', () => {
```

Reply:
It would be better to stick to the standard of describe blocks using "When...."

Reply:
I have fixed this and added When Keyword